src/Writer/BCP.php

```diff
@@ -142,11 +139,8 @@ private function getVersion()
         $res = $stmt->fetchAll();
         $version = $res[0][0];
 
-        if (intval($version) > 13) {
-            $version = 13;
-        }
-        if (empty($version)) {
-            $version = 12;
+        if (intval($version) > 17 || empty($version)) {
```

Comment:
Je vůbec ta detekce té verze potřeba? Moc tomu teda nerozumím https://docs.microsoft.com/en-us/sql/relational-databases/import-export/non-xml-format-files-sql-server?view=sql-server-2017#Structure
Ale nejde tam hardcodnout verze BCP která je v tom image?

Reply:
zkusil jsem tam dát 10 a jede to taky:)

Reply:
jo, to mas asi pravdu, ze by sa to tam mohlo zahardkodovat